components/registration/registration-form.tsx

```diff
@@ -0,0 +1,178 @@
+          </option>
+          <option>A</option>
+          <option>B</option>
+          <option>C</option>
+        </InputSelect>
+      </div>
+      <div className="mb-6">
+        <div className="mb-3">
+          <span>Kecamatan</span>
+        </div>
+        <InputSelect title="Urut berdasarkan">
+          <option disabled hidden selected value="">
+            -Masukkan Kecamatan-
+          </option>
+          <option>A</option>
+          <option>B</option>
+          <option>C</option>
+        </InputSelect>
+      </div>
+      <div className="mb-6">
+        <div className="mb-3">
+          <span>PKM</span>
+        </div>
+        <InputSelect title="Urut berdasarkan">
+          <option disabled hidden selected value="">
+            -Masukkan PKM-
+          </option>
+          <option>A</option>
+          <option>B</option>
+          <option>C</option>
+        </InputSelect>
+      </div>
+      <div className="mb-6">
+        <div className="mb-3">
+          <span> Instansi Asal</span>
+        </div>
+        <div>
+          <InputCheckboxCustom className="mr-4 w-3/12" name="instance">
+            TNI
+          </InputCheckboxCustom>
+          <InputCheckboxCustom className="mr-4 w-3/12" name="instance">
+            Polri
+          </InputCheckboxCustom>
+          <InputCheckboxCustom className="w-3/12" name="instance">
+            Puskesmas
+          </InputCheckboxCustom>
+        </div>
+      </div>
+      <div className="mb-6">
+        <div className="mb-2">
+          <span>Nama ID / Username</span>
+        </div>
+        <div className="text-red-500 italic text-sm mb-2">
+          <div className="flex">
+            <div className="mr-1">
+              <InformationCircleIcon className="w-5 h-5" />
+            </div>
+            <div>
+              <b>Nama ID</b> ini akan digunakan untuk login. Harap di ingat!
+            </div>
+          </div>
+        </div>
+        <InputText placeholder="Masukkan Nama ID" />
+        <div className="mt-2">
+          <span className="text-gray-500 ">* Tidak Boleh ada spasi minimal 5 karakter</span>
+        </div>
+      </div>
+      <div className="mb-6">
+        <div className="mb-2">
+          <span>Kata Sandi / Password</span>
+        </div>
+        <div className="text-red-500 italic text-sm mb-2">
+          <div className="flex">
+            <div className="mr-1">
+              <InformationCircleIcon className="w-5 h-5" />
+            </div>
+            <div>
+              <b>Kata Sandi </b> ini akan digunakan untuk login. Harap di ingat!
+            </div>
+          </div>
+        </div>
+        <div className="mt-1 relative rounded-md shadow-sm">
+          {/* <input
+            type="text"
+            name="price"
+            id="price"
+            className="focus:ring-indigo-500 focus:border-indigo-500 block w-full pl-7 pr-12 sm:text-sm border-gray-300 rounded-md"
+            placeholder="0.00"
+          /> */}
```

Comment:
```suggestion
```

Please don't leave any commented code behind.

Reply:
Ahh noted, i forgot to remote it..